### ISSUE TITLE
Support folder prefix 'paths' in tsconfig.json

### ISSUE THREAD

**Issue by andliang** [Azure/azure-functions-host#8767]
#### What problem would the feature you're requesting solve? Please describe.
TLDR;
I would like to import private typescript modules using the prefix paths defined in `tsconfig.json`. Attempting to use prefix paths results in "Cannot find module" errors (see below).

**More details...**
So for example, in my typescript code I would like to reference a private class (eg: ConfigManager) such as so

`import { ConfigManager } from "CompanyName/Common/ConfigManager"`

This compiles just fine when I run `tsc` and with the following setting in `tsconfig.json`
`"paths": {      
      "CompanyName/*": ["src/*", "tests/*"]     
    }`
 
But when execute `func start` in command line, I get the following error:

`Exception: Worker was unable to load function src: 'Error: Cannot find module 'CompanyName/Common/ConfigManager'`

#### Describe the solution you'd like
I would like Azure Functions to respect the  prefix `paths` in tsconfig.json for Node applications. 

#### Describe alternatives you've considered
As a workaround, I can import modules using relative paths (eg: "../../ConfigManger") but that's not an option that I would like to entertain because relative paths can be hard to understand and read, esp when they get very long.

**Comment by ramya894** [comment]
@andliang Please refer the below link for solution. 
https://stackoverflow.com/questions/43281741/how-to-use-paths-in-tsconfig-json
 If you still face the issue provide us the functionapp name, innvocation ID or timestamp details so that we can check for the logs and provide with more insights.
